### PR TITLE
Fix for log level so it isn't always DEBUG

### PR DESCRIPTION
**Pull request by pj-linebaugh**
The DEBUG env variable was always set to "True".  Added a way to set that.  In order for this to work a new appstore image is also needed.

### CODE REVIEW

**Review by joshua-seals — APPROVED** [review]
Did you mean to hard code your image in "pjl-202301231947" values.yaml tag ?

File: values.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ image:
   # -- pull policy
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag:
+  tag: pjl-202301231947
```
Comment (Hoid) [inline values.yaml:13]: is this intentional? I don't think it should be

Comment (pj-linebaugh) [reply values.yaml:13]: Yes.  Need to set it to some image that exists.  that doesn't work as appVersion.

Comment (pj-linebaugh) [reply values.yaml:13]: I have a PR in appstore for another fix and it built an image for that.  I'll add a tag and put that in appVersion.

Comment (pj-linebaugh) [reply values.yaml]: The appstore PR was merged, so I changed the tag to 'develop'.

**Pull request merged**